modular_skyrat/modules/mob_container/mob_container.dm

```diff
@@ -0,0 +1,36 @@
+/mob/proc/store_mob(mob/tostore)
+	if(!istype(tostore))
+		return
+
+	if(!tostore.on_enter_container(src) || !on_container_entered(tostore))
+		return
+
+	ADD_TRAIT(tostore, TRAIT_OXYIMMUNE, REF(src)) // We give the mob inside oxy-damage immunity. Hope this doesn't come back to bite us in the ass.
```

Comment:
:/ power crep

Reply:
extreme role play mechanics don't need balance.

Reply:
yes they do

Reply:
> extreme role play mechanics don't need balance.

Nah mate we don't need shit like the leather whip being a worse police baton

Reply:
> > extreme role play mechanics don't need balance.
> 
> Nah mate we don't need shit like the leather whip being a worse police baton

Doesn't that make it far easier to single out bad actors tho?

Reply:
no